plugins/modules/aci_cloud_subnet.py

```diff
@@ -36,7 +37,7 @@
     description:
     - Ip address of the Cloud Subnet.
     type: str
-    aliases: [subnet]
+    aliases: [ip]
```

Comment:
should we keep ip and subnet?

Reply:
`subnet` is already used for name alias and is what's defined in the argument spec. All I needed was to move subnet to match the spec but I thought adding `ip` to `subnet` would be good.

Reply:
True, the alias names may be a little off originally. However, I didn't want to make any breaking changes.